src/database/adatabase.h

```diff
@@ -88,6 +88,23 @@ class ADatabaseError : public QSqlError {
  */
 class ADatabase : public QObject {
     Q_OBJECT
+public:
+
+    using QueryResult_qstr = QString;
+    using QueryResult_i = int;
+
+    using PilotName_qstr = QueryResult_qstr;
+    using PilotRowId_i = QueryResult_i;
+
+    using TailRgstr_qstr = QueryResult_qstr;
+    using TailId_i = QueryResult_i;
+
+    using ArprtICAO_qstr = QueryResult_qstr;
+    using ArprtIATA_qstr = QueryResult_qstr;
+    using ArprtName_qstr = QueryResult_qstr;
+    using ArprtId_i = QueryResult_i;
```

Comment:
`using RowId = int` for ArprtId_i, TailId_i, PilotRowId_i
`using PilotName = QString;`
`using AirportICAO = QString;`
`using AirportIATA = QString;`
`using AirportName = QString;`
`using TailRegistration = QString;`

Reply:
These and the previous comment have a discrepency. Should the RowId or the int be used in the QMap?

Reply:
I would go for RowId